src/services/job_service.py

```diff
@@ -24,15 +22,13 @@
 #  can view the job entity to also run and delete the job.
 from services.job_handler_interface import Job, JobHandlerInterface, JobStatus
 from services.job_scheduler import scheduler
-from utils.get_extends_from import get_extends_from
 from utils.logging import logger
 from utils.string_helpers import split_absolute_ref
 
-
-def is_cron_job(blueprint_ref: str) -> bool:
-    """Checks if the type SIMOS.CRONJOB is in the list of types the blueprint extends from"""
-    all_extends = get_extends_from(blueprint_ref)
-    return SIMOS.CRON_JOB.value in all_extends
+# def is_cron_job(blueprint_ref: str) -> bool:
+#     """Checks if the type SIMOS.CRONJOB is in the list of types the blueprint extends from"""
+#     all_extends = get_extends_from(blueprint_ref)
+#     return SIMOS.CRON_JOB.value in all_extends
```

Comment:
I guess this code should be included later? If so, we can add a TODO here:

TODO add cron job support
or something